src/main/kotlin/step2/domain/WinningAmount.kt

```diff
@@ -0,0 +1,8 @@
+package step2.domain
+
+enum class WinningAmount(var label: String, var value: Int) {
```

Comment:
```suggestion
enum class WinningAmount(val label: String, val value: Int) {
```
이곳도 val로 충분할 것 같습니다 🙂